CONTRIBUTING.md

```diff
@@ -141,6 +150,9 @@ pytest
 
 # Run all the unit tests defined in the specified test file.
 pytest tests/${TARGET_TEST_FILE_NAME}
+
+# Run the unit test function with the specified name defined in the specified test file.
+pytest tests/${TARGET_TEST_FILE_NAME} -k ${TARGET_TEST_FUNCTION_NAME}
```

Comment:
Since these lines explain how to run a specific test function, the L145 sentence seems to be inconsistent. I think we can remove `all` from it. How about updating it as well?